js/lib/urlutil.js

```diff
@@ -346,11 +347,22 @@ const UrlUtil = {
         return query.file
       }
     }
-    return url.replace(`chrome-extension://${pdfjsExtensionId}/`, '')
+    return UrlUtil.getUrlFromPDFUrl(url)
+  },
+
+  isUrlPDF: function (url) {
+    return !(!url || url.indexOf(pdfjsBaseUrl) === -1)
```

Comment:
minor: change this to `url && url.indexOf(pdfjsBaseUrl) === 0` so it's more precise and easier to read

Reply:
fixed